tests/sls-api-file-list.js

```diff
@@ -0,0 +1,310 @@
+'use strict';
+
+const glabalSandbox = require('sinon').createSandbox();
+const APITest = require('@janiscommerce/api-test');
+const { ApiListData } = require('@janiscommerce/api-list');
+const { SlsApiFileList } = require('../lib/index');
+const BaseModel = require('../lib/base-model');
+
+describe('File List Api', () => {
+	const apiExtendedSimple = () => {
+		class API extends SlsApiFileList {}
+		return API;
+	};
+
+	afterEach(() => {
+		glabalSandbox.restore();
+	});
+
+	beforeEach(() => {
+		glabalSandbox.stub(ApiListData.prototype, '_getModelInstance').returns(new BaseModel());
+	});
+
+	const rowGetted = {
+		id: 20,
+		name: 'what.jpeg',
+		claimId: 7,
+		size: 5014,
+		mimeType: 'image/png',
+		path: 'file/1233.jpg',
+		type: 'image',
+		dateCreated: 1576269240,
+		userCreated: 5,
+		dateModified: 1576272801,
+		userModified: null
+	};
+
+	const rowGetted2 = {
+		id: 21,
+		name: 'file.txt',
+		claimId: 7,
+		size: 8405,
+		mimeType: 'image/png',
+		path: 'file/1233.txt',
+		type: 'other',
+		dateCreated: 1576269240,
+		userCreated: 5,
+		dateModified: 1576272801,
+		userModified: null
+	};
+
+	const rowFormatted = {
+		id: 20,
+		name: 'what.jpeg',
+		claimId: 7,
+		size: 5014,
+		mimeType: 'image/png',
+		type: 'image',
+		dateCreated: 1576269240,
+		userCreated: 5,
+		dateModified: 1576272801,
+		userModified: null
+	};
+
+	const rowFormatted2 = {
+		id: 21,
+		name: 'file.txt',
+		claimId: 7,
+		size: 8405,
+		mimeType: 'image/png',
+		type: 'other',
+		dateCreated: 1576269240,
+		userCreated: 5,
+		dateModified: 1576272801,
+		userModified: null
+	};
+
+	APITest(apiExtendedSimple(), [
+		{
+			before: sandbox => {
+				sandbox.stub(BaseModel.prototype, 'get')
+					.returns([]);
```

Comment:
Todo lo que son modelo, resuelve, no devuelve.

```suggestion
					.resolves([]);
```